src/lib/resolver/resolver.factory.ts

```diff
@@ -97,6 +100,20 @@ function addDeclarationToModule(options: ProviderOptions): Rule {
       options.module,
       declarator.declare(content, options as DeclarationOptions),
     );
+
+    const DependencyRef = getPackageJsonDependency(
+      tree,
+      '@nestjs/graphql',
```

Comment:
Where do we require graphql?

Reply:
Idk, it was mentioned already so made the provisions for imports.